packages/client/webpack.config.js

```diff
@@ -21,6 +21,14 @@ module.exports = {
         test: /\.js$/,
         loader: 'babel',
         exclude: /node_modules/,
+        query: {
+          babelrc: false,
```

Comment:
Because we need a different config with different plugins and presets for CommonJS build